etc/linux-desktop/syncthing-start.desktop

```diff
@@ -2,7 +2,7 @@
 Name=Start Syncthing
 GenericName=File synchronization
 Comment=Starts the main syncthing process in the background.
-Exec=/usr/bin/syncthing -no-browser
+Exec=/usr/bin/syncthing -no-browser -logfile=~/.config/syncthing/syncthing.log
```

Comment:
You're going to unleash the fucking *fury* of the XDG aficionados here, be prepared to defend yourself :D

Reply:
I know (and I even have more affections than aversion towards XDG myself (I definitely prefer it a thousand times to programs cluttering my home with `.hereIAm` dirs at the other end of the extremes), so I might eventually get around to fixing https://github.com/syncthing/syncthing/issues/4924, including log location) :)